presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveModule.java

```diff
@@ -59,5 +59,6 @@ public void setup(Binder binder)
         binder.bind(IcebergMetadataFactory.class).to(IcebergHiveMetadataFactory.class).in(Scopes.SINGLETON);
 
         configBinder(binder).bindConfig(MetastoreConfig.class);
+        binder.bind(IcebergSessionProperties.class).in(Scopes.SINGLETON);
```

Comment:
nits: we can remove this, since it has been bound in `IcebergCommonModule`.

Reply:
Thank you for catching this and for two very thoughtful review 🙂